pandas/errors/__init__.py

```diff
@@ -395,3 +395,21 @@ def __init__(self, message: str) -> None:
         # attr only exists on Windows, so typing fails on other platforms
         message += f" ({ctypes.WinError()})"  # type: ignore[attr-defined]
         super().__init__(message)
+
+
+class CSSWarning(UserWarning):
+    """
+    Warning is raised when trying to convert styling to a css format and it failing.
```

Comment:
Fails instead of and it failing?

Reply:
Ah yeah, my bad. I've reworded the sentence. Let me know what you think